pkgs/tools/misc/vmtouch/default.nix

```diff
@@ -2,14 +2,14 @@
 
 stdenv.mkDerivation rec {
   pname = "vmtouch";
-  version = "git-20150310";
+  version = "1.0.2";
```

Comment:
I agree with you, that it is strange, but that's the only way to keep backwards compatibility. Or wait for nix to have feature to parse `git` part..